lib/core/core.cmake

```diff
@@ -0,0 +1,17 @@
+set (KOTEKAN_CORE_SOURCES
```

Comment:
I see why you are breaking these files up `CMakeLists.txt` plus `core.cmake`, etc.  for the auto-formatter, but I don't like the idea of having to remember or lookup which file in the folder actually contains the sources when I need to add too it.    Maybe name this file `sources.cmake` in every folder to make it standard.  Or just apply the auto-formatter to every .cpp file you find recursively?